src/main/kotlin/edu/agh/susgame/dto/rest/games/model/Result.kt

```diff
@@ -2,23 +2,26 @@ package edu.agh.susgame.dto.rest.games.model
 
 import edu.agh.susgame.dto.rest.ApiResult
 import edu.agh.susgame.dto.rest.model.GameMapDTO
-import edu.agh.susgame.dto.rest.model.Lobby
+import edu.agh.susgame.dto.rest.model.LobbyDetails
 import edu.agh.susgame.dto.rest.model.LobbyId
+import edu.agh.susgame.dto.rest.model.LobbyRow
 import java.net.HttpURLConnection
 
 
 sealed class GetAllGamesApiResult(responseCode: Int) : ApiResult(responseCode) {
-    data class Success(val lobbies: List<Lobby>) : GetAllGamesApiResult(HttpURLConnection.HTTP_OK)
+    data class Success(val lobbies: List<LobbyRow>) : GetAllGamesApiResult(HttpURLConnection.HTTP_OK)
 
     data object Error : GetAllGamesApiResult(HttpURLConnection.HTTP_INTERNAL_ERROR)
 }
 
 
 sealed class GetGameApiResult(responseCode: Int) : ApiResult(responseCode) {
-    data class Success(val lobby: Lobby) : GetGameApiResult(HttpURLConnection.HTTP_OK)
+    data class Success(val lobby: LobbyDetails) : GetGameApiResult(HttpURLConnection.HTTP_OK)
 
     data object DoesNotExist : GetGameApiResult(HttpURLConnection.HTTP_NOT_FOUND)
 
+    data object InvalidPin : GetGameApiResult(HttpURLConnection.HTTP_UNAUTHORIZED)
```

Comment:
po co to? do pin podaje się w /join więc dostaniesz serversocketerror i połączenie się zamknie

Reply:
Relna różnica pomiędzy `GetAll`, a `getSingle` z perspektywy klienta jest taka, że gdy dołączymy do lobby, to widzimy listę graczy, a nie tylko liczbę. Uznałem, że taki flow ma sens, że żeby zobaczyć nicki graczy, trzeba znać PIN

Reply:
Poza tym jest to kompatibilne z flow-em, który Łukasz już naklepał

Reply:
chodzi mi o to że `InvalidPin` nie będzi enigdy zwrócone przez backend

Reply:
jak wam to nie przeszkadza to spoko